zaza/openstack/charm_tests/glance_simplestreams_sync/setup.py

```diff
@@ -77,3 +78,15 @@ def sync_images():
         logging.info('Contents of Keystone service catalog: "{}"'
                      .format(pprint.pformat(catalog)))
         raise
+
+
+def set_latest_property_config():
```

Comment:
Any particular reason this is a setup method rather than just an option in one of the test bundles?

Reply:
my change doesn't add the `latest=true` property to existing images, just for the newly-created.
also, I couldn't find a way to force a new sync of images on top of the existing one, and deleting the image and re-sync seemed too extreme.
thus I need to set the config before the sync actually happen. 

@ajkavanagh asked the same thing [here in this collapsed comment](https://github.com/openstack-charmers/zaza-openstack-tests/pull/853#discussion_r938714958)

Reply:
@wolsen and @ajkavanagh 
my change has been [merged on simplestreams repo](https://git.launchpad.net/simplestreams/commit/?id=bc0d94320bb6ce56c7ec0171bcf087f4eb921e87). how can I reference this commit to the CI tests?